api/src/pcapi/routes/serialization/finance_serialize.py

```diff
@@ -53,3 +60,65 @@ def from_orm(cls, invoice: finance_models.Invoice) -> "InvoiceResponseModel":
 
 class InvoiceListResponseModel(BaseModel):
     __root__: list[InvoiceResponseModel]
+
+
+class LinkedVenues(BaseModel):
+    """A venue that is already linked to a bank account."""
+
+    id: int
+    common_name: str
+
+    class Config:
+        orm_mode = True
+
+
+class ManagedVenues(BaseModel):
+    id: int
+    common_name: str
+    siret: str
+    bankAccountId: int | None
+
+    class Config:
+        orm_mode = True
+
+    @classmethod
+    def from_orm(cls, venue: "offerers_models.Venue") -> "ManagedVenues":
+        """
+        For each managed venue, we only want to display the actual bank account
```

Comment:
Tu veux probablement dire : 
```suggestion
        For each managed venue, we only want to display the current bank account
```